book/src/remapping.md

```diff
@@ -11,6 +11,7 @@ this:
 ```toml
 # At most one section each of 'keys.normal', 'keys.insert' and 'keys.select'
 [keys.normal]
+C-s = ":w" # Maps the Control-s to the typable command :w which is an alias for :write (save file)
```

Comment:
Let's add another example that uses arguments, for example `:open ~/.config/helix/config.toml`